doc/19-technical-concepts.md

```diff
@@ -632,16 +632,19 @@ The algorithm works like this:
  * Set the authority (true or false)
 
 The object authority calculation works "offline" without any message exchange.
-Each instance alculates the SDBM hash of the config object name, puts that in contrast
-modulo the connected endpoints size.
+Each instance calculates the SDBM hash of the config object name up to the
+host name separator "!", puts that in contrast modulo the connected endpoints size.
```

Comment:
Not all Icinga 2 objects are bound to a specific host, so please describe this clearly and not just extend the existing sentence. Apart from that, can you please adjust this sentence (yes you didn't introduce it but still it doesn't reflect the current implementation).
> * If there's two endpoints, but only us seeing ourselves and the application start is less than 60 seconds in the past, do nothing (wait for cluster reconnect to take place, grace period).

Specifically, it states _less than **60** seconds_ but the current used value is `30s` changed with https://github.com/Icinga/icinga2/commit/149f640fd8e9191efcf69a60e8ce83f38101d359.

Reply:
The latter is definitely out of this PR's scope (just like #10441). And yes, "!" is our _host name separator_. 🤷‍♂️ Otherwise one could wonder: why "!"?

Reply:
> The latter is definitely out of this PR's scope (just like #10441)

It's y _2_ char/number fix, why opening a PR just for this?

> And yes, "!" is our _host name separator_. 🤷‍♂️ Otherwise one could wonder: why "!"?

I didn't meant that with my comment! Are all Icinga 2 objects a child of some host? No, and that's why you're conditionally erasing the string after the `"!"` char, thus instead of changing the existing sentence and making it basically sound as if all objects were bound to some host, add extra sentence explaining the behaviour specific to host and its child objects.

Reply:
Because it's merged faster than this far more complex PR even gets approved. And there's no dependency.